src/serializer/gtfx_flex_serializer.py

```diff
@@ -0,0 +1,90 @@
+import json
+import datetime
+import decimal
+
+
+class Jsonable(object):
```

Comment:
Can we use modeller similar to the one in typescript with @Prop ??

Reply:
I try to make it similar to @props